systest/backup/advanced-scenarios/127-Namespace/backup_test.go

```diff
@@ -0,0 +1,91 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+	"testing"
+
+	"github.com/dgraph-io/dgraph/systest/backup/common"
+	"github.com/dgraph-io/dgraph/testutil"
+)
+
+var (
+	restoreLocation = "/data/backups/"
+	backupDst       = "/data/backups/"
+	localBackupDest = "./data/backups"
+	headerAlpha1    = http.Header{}
+	headerAlpha2    = http.Header{}
+)
+
+const (
+	accessJwtHeader = "X-Dgraph-AccessToken"
+)
+
+func TestDeletedNamespaceID(t *testing.T) {
+	err := common.RemoveContentsOfPerticularDir(t, "./data/backups/")
```

Comment:
can we just delete the dir and recreate it?

Reply:
We are using the './data/backups' directory as a 'volume bind' in docker-compose.yml. Deleting this folder will cause problems while bringing up the cluster.